### PR TITLE
zellij: Housekeeping

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Add appstream metainfo (Part of getsolus/packages#1389)
- Use cargo macros (Part of getsolus/packages#3111)

**Test Plan**

Verify metainfo with appstream-builder --packages-dir=. --include-failed -v

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Review by algent-al — APPROVED** [review]
LGTM, thanks!

File: packages/z/zellij/files/dev.zellij.Zellij.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```
Comment (EbonJaeger) [inline packages/z/zellij/files/dev.zellij.Zellij.metainfo.xml:2]: zellij is a terminal application; is this the right component?

Comment (malfisya) [reply packages/z/zellij/files/dev.zellij.Zellij.metainfo.xml:2]: Oh, I thought it was more like `kitty`. Now that i tried it, it is more like `helix`. Yeah, this is not the right component. But then again, it provides desktop file which console application are not required to. For console application, [this](https://www.freedesktop.org/software/appstream/metainfocreator/#/consoleapp) the template for it.

Comment (malfisya) [reply packages/z/zellij/files/dev.zellij.Zellij.metainfo.xml:2]: I think the component can be changed to `<component type="console-application">` and will still be valid. Let me try in a sec.

**Pull request merged**